client/src/components/gallery/Strip.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import styled from 'styled-components'
+
+const Wrapper = styled.div`
+  margin-top: 4px;
+  position: relative;
+  height: 69px;
+  margin-bottom: 4px;
+  z-index: 40;
+`;
+const StripScroll = styled.div`
+  left: ${props => props.stripPos};
+  width: 1562px;
+  position: absolute;
+`;
+
+const HighlightSelector = styled.div`
+  position: absolute;
+  left: ${props => props.selectorPos}
+  background-image: url(https://store.steampowered.com/public/images/v5/game_highlight_activethumb.png);
+  width: 116px;
+  height: 77px;
+  z-index: 50;
+  top: -10px;
+  pointer-events: none;
+`;
+const Screenshot = styled.div`
+  height: 65px;
+  width: 116px;
+  float: left;
+  cursor: pointer;
+  text-align: center;
+  margin: 2px;
+  background-color: #000000;
+  position: relative;
+`;
+
+const MovieMaker = styled.div`
+  position: absolute;
+  top: 16px;
+  left: 42px;
+  width: 32px;
+  height: 32px;
+  background-image: url(https://store.steampowered.com/public/images/v5/ico_game_highlight_video.png);
+`;
+
+const Img = styled.img`
+  width: 115px;
+  height: 65px
+  border: none;
+`;
+class Strip extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+  
+  render() {
+    var stripPos = String(this.props.stripPos) + 'px';
+    var selectorPos = String(this.props.selectorPos) + 'px';
+    var firstVideo = null;
+    var secondVideo = null;
+    var lastVideo = null;
+    if(this.props.videos) {
+      firstVideo = 
```

Comment:
Are there only ever 3 videos at most? Why are you hard coding array index instead of using Array.map()?

Reply:
Yes, there are at most three videos. Cause screenshots cut the order of videos. There are two in front of 10 screenshots and one after them.